packages/server/__snapshots__/6_visit_spec.coffee.js

```diff
@@ -123,7 +123,7 @@ The stack trace for this error is:
 Error: connect ECONNREFUSED 127.0.0.1:16795
     [stack trace lines]
 
-  
+      [stack trace lines]
```

Comment:
These all don't look right.

Reply:
Actually, I think these are fine because the error message includes the original errors' stack trace and then we have our own stack trace now that points to where this occurred in the user's spec file.

The entire error looks like this:

```
CypressError: `cy.visit()` failed trying to load:

http://localhost:16795/

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: connect ECONNREFUSED 127.0.0.1:16795

Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer

The stack trace for this error is:

Error: connect ECONNREFUSED 127.0.0.1:16795
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1:1)

      at Context.eval (http://localhost:51748/__cypress/tests?p=cypress/integration/visit_http_network_error_failing_spec.coffee:4:15)
```